src/views/ApartmentApp/components/StaffMenu/components/ApplicationTable/components/ApplicationRow/components/ApplicantSubTable/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Toolbar,
+  Typography,
+} from '@material-ui/core/';
+
+/**
+ * @typedef { import('../../../../../../../../../../services/housing').ApartmentApplicant } ApartmentApplicant
+ */
+
+/**
+ * Renders the applicant sub-table for the apartment application staff page
+ * @param {Object} props The React component props
+ * @param {ApartmentApplicant[]} props.applicants Array of applicant info
+ * @returns {JSX.Element} JSX Element for the applicant sub-table
+ */
+const ApplicantSubTable = ({ applicants }) => {
+  const [thisYear, setThisYear] = useState();
+
+  useEffect(() => {
+    const getYear = () => setThisYear(new Date().getFullYear());
+
+    getYear();
```

Comment:
It really doesn't matter, but this could be simplified:
```suggestion
    setThisYear(new Date().getFullYear());
```

Reply:
Thank you, I don't know I did it that way...
I approve this for commit